test/test_torch.py

```diff
@@ -3461,19 +3456,19 @@ def test_bool_tensor_value_change(self):
             x[1] = True
             self.assertEqual(x, torch.tensor([False, True], dtype=torch.bool))
 
-    def test_unfold_all_devices_and_dtypes(self):
-        for device in torch.testing.get_all_device_types():
-            for dt in torch.testing.get_all_dtypes():
-                if dt == torch.bfloat16:
-                    self.assertRaises(RuntimeError, lambda: torch.randint(5, (0, 1, 3, 0), dtype=dt, device=device))
-                    continue
+    @pytorchtest.test_all_device_types()
```

Comment:
nit: can we call this `torchtest`?  We don't actually use the name "PyTorch" much in the code...